app/javascript/components/mentoring/mentor_conversations_list/pagination.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+
+export default function Pagination({ current, total, setPage, around = 3 }) {
```

Comment:
Consider **not** using `default` exports. Named exports do _waaay_ better with intellisense.

Reply:
WIll do. Thanks!